### PR TITLE
RDK-36160 : Smart audio mixing and volume control in SAP

### PR DESCRIPTION
**Pull request by muthumanik**
Reason for change: Smart audio mixing and volume control in SAP
Test Procedure: None
Risks: None
Signed-off-by: muthumani.tsm@gmail.com

Change-Id: Ib234add22bff211a6836d1f7ef76d3ea3b70e555

### CODE REVIEW

File: docs/api/SystemAudioPlayerPlugin.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ SystemAudioPlayer interface methods:
 | [playbuffer](#playbuffer) | Buffers the audio playback on the specified player |
 | [resume](#resume) | Resumes playback on the specified player |
 | [setMixerLevels](#setMixerLevels) | Sets the audio level on the specified player |
+| [setSmartVolControl](#setSmartVolControl) | Sets the smart audio volume level on the specified player |
```
Comment (anand-ky) [inline docs/api/SystemAudioPlayerPlugin.md:99]: These docs are autogenerated. These should not be manually edited. You would need to change the SystemAudioPlayer.json file and add the API there.

**Review by anand-ky — CHANGES_REQUESTED** [review]
Pls also sign CLA (Contributor License Agreement)

File: docs/api/SystemAudioPlayerPlugin.md (hunk 1 above)
Comment (muthumanik) [reply docs/api/SystemAudioPlayerPlugin.md:99]: Hi Anand, As you suggest , updated the new api information added as part of SystemAudioPlayer.json

**Comment by muthumanik** [comment]
Facing issues with cla signed. Recommended to create new pull request.

**Pull request closed without merging**